packages/material/src/layouts/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './group.layout';
+export * from './material-group.layout';
```

Comment:
This doesn't fit with the deletion of the group.layout and the rename of material-group.layout to MaterialGroupLayout